### PR TITLE
CFME 4.2 Update

### PR DESCRIPTION
**Pull request by lpsantil**
#### What is this PR About?
Adds notes about CFME 4.2 integration and chargeback data delays.

#### How should we test or review this PR?
Integrate a CFME 4.2 with OCP 3.3+ cluster.

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

File: playbooks/operationalizing/cloudforms.adoc [hunk 1]
```diff
@@ -223,6 +223,35 @@ Under *Endpoints*, select the *Hawkular* tab and enter the hawkular-metrics host
...
 
+==== CloudForms 4.2:
+
+In the CloudForms web console, hover over *Compute* in the menu on the left, then *Containers*, then select *Providers*.
+
+Select the *Configuration* button and then *Add a New Containers Provider*.
+
+Enter a name for the provider and select *OpenShift Enterprise* in the type (ignore the zone, this will populate on its own). 
+
+Under *Endpoints*, in the *Default* tab, enter the hostname or address of the HAProxy server (or the master in a single master setup) into the *Hostname* field and enter the token retrieved earlier into the *Token* and *Confirm Token* fields. Click *Validate* to confirm connectivity.
```
Comment (sabre1041) [inline playbooks/operationalizing/cloudforms.adoc]: HAProxy is not the OpenShift router correct, but an entry fronting the Master API? If so, I would rephrase and remove HAProxy as it could be confusing

Comment (sabre1041) [inline playbooks/operationalizing/cloudforms.adoc]: My assumption is that this is for the Master API and not the HAProxy router. If so, we should probably rephrase to mention the more generic load balancer in front of the masters.

Since it is my assumption that you used the 4.1 instructions as a base, if you make changes here, would you be able to replicate for the 4.1 instructions

File: playbooks/operationalizing/cloudforms.adoc [hunk 2]
```diff
@@ -223,6 +223,35 @@ Under *Endpoints*, select the *Hawkular* tab and enter the hawkular-metrics host
...
+
+Click *Add* to add the container provider.
+
+In the upper-right hand corner, click on the dropdown next to your *username* and then click on the *Configuration* menu item.  
+
+In the *Settings* tree, click on the *CFME Region*, then click on *Zones*, the OCP *Zone*, and finally pick the *Server*.
+
+Under the *Server* tab, in the *Server Control* section, under *Server Roles*, turn on the checkboxes *Capacity & Utilization Coordinator*, *Capacity & Utilization Data Collector*, *Capacity & Utilization Data Processor*.
+
+Capacity and Utilization metrics will begin to flow into CFME and will take approximately 24 hours to process data and make the default report generation and visualizations available.  For chargeback reports, it takes approximately a total of 48 hours for the report data to become available.  Additionally, chargeback data is not available for the current day (only prior day, 24 hour lag) because it has not been analyzed yet.
```
Comment (sabre1041) [inline playbooks/operationalizing/cloudforms.adoc]: Since there is a time delay between configuration and when data first appears, I would move that information to a callout so that it has greater visibility

**Comment by etsauer** [comment]
@lpsantil any updates on this? would love to get it merged.

File: playbooks/operationalizing/cloudforms.adoc (hunk 1 above)
Comment (lpsantil) [inline playbooks/operationalizing/cloudforms.adoc]: I agree.

Comment (lpsantil) [inline playbooks/operationalizing/cloudforms.adoc]: I can take a crack at it.

File: playbooks/operationalizing/cloudforms.adoc (hunk 2 above)
Comment (lpsantil) [inline playbooks/operationalizing/cloudforms.adoc]: Will do.

**Comment by lpsantil** [comment]
@etsauer Apologies for not getting to this sooner.  Transitioning to new project, blah, blah.  I will work on the recommendations/suggestions this week.

File: playbooks/operationalizing/cloudforms.adoc (hunk 2 above)
Comment (lpsantil) [inline playbooks/operationalizing/cloudforms.adoc]: Updated.

Comment (lpsantil) [inline playbooks/operationalizing/cloudforms.adoc]: Updated.

Comment (lpsantil) [inline playbooks/operationalizing/cloudforms.adoc]: Ok.  Updated.

File: playbooks/operationalizing/cloudforms.adoc [hunk 3]
```diff
@@ -231,7 +231,7 @@ Select the *Configuration* button and then *Add a New Containers Provider*.
 
 Enter a name for the provider and select *OpenShift Enterprise* in the type (ignore the zone, this will populate on its own). 
 
-Under *Endpoints*, in the *Default* tab, enter the hostname or address of the HAProxy server (or the master in a single master setup) into the *Hostname* field and enter the token retrieved earlier into the *Token* and *Confirm Token* fields. Click *Validate* to confirm connectivity.
+Under *Endpoints*, in the *Default* tab, enter the cluster public hostname (openshift_master_cluster_public_hostname in `/etc/ansible/hosts`) into the *Hostname* field and enter the token retrieved earlier into the *Token* and *Confirm Token* fields. Click *Validate* to confirm connectivity.
```
Comment (sabre1041) [inline playbooks/operationalizing/cloudforms.adoc]: @lpsantil I would simply say "enter the address of the OpenShift Master API into the...". The level of detail could confuse some users. Also, In most cases, users will not modify the default Ansible host file but create a separate file for each OpenShift environment

**Comment by sabre1041** [comment]
@lpsantil added another comment regarding one of the corrections. Otherwise, it looks good

**Comment by lpsantil** [comment]
Updated.

**Comment by sabre1041** [comment]
@lpsantil LGTM

@JaredBurck want to take a look?
